### PR TITLE
[shopsys] updated codeception to version 5

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| We want to keep our dependencies as up to date as possible
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| Yes 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Review by LuKillman — APPROVED** [review]

File: open-source-license-acknowledgements-and-third-party-copyrights.md [hunk 1]
```diff
@@ -92,9 +92,9 @@ License: Creative Commons Attribution 3.0 BY
 https://github.com/erikdubbelboer/phpRedisAdmin/blob/master/README.markdown
 
 ### Selenium Docker
-Image: `Selenium/standalone-chrome:3.11`  
-License: Apache License 2.0  
-https://github.com/SeleniumHQ/docker-selenium/blob/master/LICENSE.md
```
Comment (grossmannmartin) [inline open-source-license-acknowledgements-and-third-party-copyrights.md:97]: two spaces are mandatory to preserve formatting 

now it looks like this
Snímek obrazovky 2023-05-05 v 19 14 31

File: project-base/ecs.php [hunk 2]
```diff
@@ -99,6 +100,9 @@
             __DIR__ . '/tests/App/Test/Codeception/Helper/SymfonyHelper.php',
             __DIR__ . '/tests/App/Test/Codeception/Module/Db.php',
         ],
+        EmptyStatementSniff::class => [
```
Comment (grossmannmartin) [inline project-base/ecs.php:103]: is it necessary? I don't see any empty statements in this class

**Review by grossmannmartin — CHANGES_REQUESTED** [review]

File: open-source-license-acknowledgements-and-third-party-copyrights.md (hunk 1 above)
Comment (TomasLudvik) [reply open-source-license-acknowledgements-and-third-party-copyrights.md:97]: I forgot to mention to check only the last commit as it is only one different from the previous PR, but I have fixed it there so it is already fixed here

File: packages/framework/tests/Test/Codeception/ActorInterface.php [hunk 3]
```diff
@@ -6,55 +6,63 @@
...
-     * @param string $databaseKey
+     * @param string $routeName
+     * @param array $parameters
      */
-    public function amConnectedToDatabase($databaseKey);
+    public function amOnLocalizedRoute(string $routeName, array $parameters = []): void;
 
     /**
-     * @param string $page
+     * @param mixed $page
```
[the remark this answers is not in the document]
Comment (TomasLudvik) [reply packages/framework/tests/Test/Codeception/ActorInterface.php:30]: The problem is that this interface is generated from `AcceptanceTesterActions` from codeception and it depends on how they defined those types. For mixed it means there is no type defined.

**Review by LuKillman — APPROVED** [review]

**Review by pk16011990 — APPROVED** [review]

**Review by grossmannmartin — APPROVED** [review]
nice work

**Pull request merged**